apps/easypid/README.md

```diff
@@ -34,6 +34,7 @@ This app was created by [Animo Solutions](https://animo.id/) in the context of t
 
 ## Install
 
+<<<<<<< HEAD
```

Comment:
this file has several merge conflicts still in there

Reply:
Could it be that you reviewed before I sent the link? I resolved all the conflicts I think